src/main/java/mate/academy/controller/BookController.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.controller;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.dto.BookDto;
+import mate.academy.dto.CreateBookRequestDto;
+import mate.academy.service.BookService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/books")
```

Comment:
add `server.servlet.context-path=/api` to `application.properties` 
```suggestion
@RequestMapping("/books")
```